lib/ronin/code/sql/emitter.rb

```diff
@@ -51,18 +54,23 @@ class Emitter
         # @param [:single, :double] quotes
         #   Type of quotes to use for Strings.
         #
+        # @param [:unset, :mysql, :postgres, :oracle, :mssql] syntax
+        #   Syntax used during code-generation
+        #
+        # @param [:auto, String] comment
+        #   String to use as the comment when terminating injection string
+        #
         # @param [Hash{Symbol => Object}] kwargs
         #   Emitter options.
         #
         # @option kwargs [:lower, :upper, :random, nil] :case
         #   Case for keywords.
         #
-        # @option kwargs [:unset, :mysql, :postgres, :oracle, :mssql] :syntax
-        #   Syntax used during code-generation
-        #
-        def initialize(space: ' ', quotes: :single, **kwargs)
+
+        def initialize(space: ' ', quotes: :single, syntax: :unset, comment: :auto, **kwargs)
           @case   = kwargs[:case] # HACK: because `case` is a ruby keyword
-          @syntax = syntax || :unset
+          @syntax = syntax
+          @comment = comment
```

Comment:
Try to vertically align the `=` otherwise `rubocop` will complain. (Granted, `rubocop` isn't setup on `ronin-code-sql` yet, but it _will_ be.)